JetStreamDriver.js

```diff
@@ -2057,16 +2057,30 @@ let BENCHMARKS = [
         tags: ["Default", "Wasm"],
     }),
     new WasmEMCCBenchmark({
-        name: "Dart-flute-wasm",
+        name: "Dart-flute-complex-wasm",
         files: [
             "./Dart/benchmark.js",
         ],
         preload: {
-            jsModule: "./Dart/build/flute.dart2wasm.mjs",
-            wasmBinary: "./Dart/build/flute.dart2wasm.wasm",
+            jsModule: "./Dart/build/flute.complex.dart2wasm.mjs",
+            wasmBinary: "./Dart/build/flute.complex.dart2wasm.wasm",
         },
         iterations: 15,
         worstCaseCount: 2,
+        tags: ["Wasm"],
+        disabledByDefault: true,
```

Comment:
I don't think you need this and it won't do anything anymore. The "Default" tag is what decides what runs unless other options are passed.